platform/source/mbed_retarget.cpp

```diff
@@ -808,6 +968,10 @@ extern "C" int isatty(int fildes)
     } else {
         return tty;
     }
+#else
+    // Is not attached to an interactive device
+    return 1;
```

Comment:
This is okay but we could also check that fildes is stdout/stderr.

Reply:
It is not necessary if everything goes to the same output.
See https://github.com/hugueskamba/mbed-os/pull/4#discussion_r339455836 .